Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM maven:3.6-jdk-8 AS builder
+WORKDIR /app
+ENV GRPC_HEALTH_PROBE_VERSION=v0.3.2
+RUN wget -qOgrpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
+    chmod +x grpc_health_probe && \
+    git clone https://github.com/OpenNMS/grpc-server.git && \
+    cd grpc-server && \
+    git checkout -b feature/health-check origin/feature/health-check && \
+    mvn package
+
+FROM openjdk:8-jdk-slim
```

Comment:
does it not work with a more modern JDK?

Reply:
Added support for OpenJDK 11